i18n/i18nTextCollector.php

```diff
@@ -27,12 +27,20 @@
  */
 class i18nTextCollector extends Object {
 	
+	/**
+	 * Default (master) locale
+	 *
+	 * @var string
```

Comment:
The point is that it's initially set to null so it has the potential to be null or a string.